types/relay-runtime/lib/store/RelayModernQueryExecutor.d.ts

```diff
@@ -4,26 +4,41 @@ import {
     OperationTracker,
     OptimisticResponseConfig,
     PublishQueue,
+    ReactFlightPayloadDeserializer,
     SelectorStoreUpdater,
+    Store,
 } from './RelayStoreTypes';
 import { GraphQLResponse } from '../network/RelayNetworkTypes';
 import { Sink, RelayObservable } from '../network/RelayObservable';
 import { GetDataID } from './RelayResponseNormalizer';
 
+export type ActiveState = 'active' | 'inactive';
+
 export interface ExecuteConfig {
     getDataID: GetDataID;
+    treatMissingFieldsAsNull: boolean;
```

Comment:
maybe `readonly` (I'll stop saying this).  But anyway, I see the other properties in this interface aren't marked `readonly` so maybe we don't do that.